tests/.coveragerc

```diff
@@ -2,4 +2,5 @@
 omit=
     tests/*
     integration_tests/*
-		test*
\ No newline at end of file
+		test*
+    */scripts/*
```

Comment:
remove scripts of classificaton from the test coverage. This is because we cannot test out these scripts as it is not possible to load the metadata models to git.

Reply:
Please revert this change.

Reply:
Suggested test details described in https://github.com/isamplesorg/isamples_inabox/issues/215#issuecomment-1259806102

Reply:
(And most of this code will move to a new package under `isamples_metadata`.